lib/engine/game/g_1880/step/company_pending_par.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/company_pending_par'
+
+module Engine
+  module Game
+    module G1880
+      module Step
+        class CompanyPendingPar < Engine::Step::CompanyPendingPar
+          def process_par(action)
+            super
+            corporation = action.corporation
+            @log << "#{corporation.name} selects ABC building permit"
+            corporation.building_permits = 'ABC'
+          end
```

Comment:
for simplicity ABC is auto selected for the BCR, which will happen 99% of the time in games. I already fixed this and will add all the choice logic in the next PR.